components/donateform.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react'
-import { CardElement, injectStripe } from 'react-stripe-elements'
+import { CardElement } from '@stripe/react-stripe-js'
```

Comment:
for some reason i can't comment on the collapsed portion of the file so i'll leave the comment here,

First, thanks so much for doing this!!! We really appreciate community contributions.

I tried this out on the preview link and got an error "You must provide a Stripe Element or a valid token type to create a Token." I believe this is because we're trying to call stripe.createToken() with no arguments. 

Reading through this migration guide https://github.com/stripe/react-stripe-js/blob/master/docs/migrating.md it looks like we might need to add this logic:

```
// Use elements.getElement to get a reference to the mounted Element.
const cardElement = elements.getElement(CardElement);

// Pass the Element directly to other Stripe.js methods:
// e.g. createToken - https://stripe.com/docs/js/tokens_sources/create_token?type=cardElement stripe.createToken(cardElement);
```

could you add that? down in this file where previously it just has `stripe.createToken`

Reply:
Sorry for the delayed response! I'll try to get to this before the end of the week.

Reply:
looks good now, thanks @zachsnoek !